### PR TITLE
Kernel of Laplace Box

### PR DESCRIPTION
**Pull request by CherifMZ**

### CODE REVIEW

**Comment by CherifMZ** [comment]
Isn't the norm calculated in [line 18](https://github.com/clEsperanto/clij-opencl-kernels/blob/clesperanto_kernels/kernels/laplace_box.cl) supposed to be **(r.z + r.y + r.x) * 2** instead of **(r.z + r.y + r.z) * 2** ?

**Comment by CherifMZ** [comment]
I have corrected the tests so that they can test the **_Laplace formula_**.
I have put the input values and the expected ones. I also added the CPU version in which, with the Laplace formula, the input is extended by filling in the values beyond the edges with the same constant value 0.

**Comment by CherifMZ** [comment]
I have corrected the tests so that they can test the **_Laplace formula_** (they obviously still don't pass).
I have put the input values and the expected ones. I also added the CPU version in which, with the Laplace formula, the input is extended by filling in the values beyond the edges with the same constant value 0.

File: clic/src/tier1/cleLaplaceBoxKernel.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+{
+
+LaplaceBoxKernel::LaplaceBoxKernel(const ProcessorPointer & device)
+  : Operation(device, 2)
+{
+  std::string cl_header = {
+#include "cle_laplace_box.h"
+  };
+  this->SetSource("laplace_box", cl_header);
+}
```
Comment (StRigaud) [inline clic/src/tier1/cleLaplaceBoxKernel.cpp:14]: Library was updated, this need to be changed as followed:
- move the include outside the function, at the top of the file
- replace `cl_header` by `oclKernel::laplace_box`

see: [absolute kernel](https://github.com/clEsperanto/CLIc_prototype/blob/master/clic/src/tier1/cleAbsoluteKernel.cpp) for example

File: thirdparty/CMakeLists.txt [hunk 2]
```diff
@@ -4,8 +4,8 @@ set(FETCHCONTENT_BASE_DIR ${CMAKE_CURRENT_BINARY_DIR})
 
 ## Manage OpenCL Kernel Sources
 FetchContent_Declare(OpenCL_Kernels
-    GIT_REPOSITORY https://github.com/clEsperanto/clij-opencl-kernels.git
-    GIT_TAG        clesperanto_kernels
+    GIT_REPOSITORY https://github.com/CherifMZ/clij-opencl-kernels.git
+    GIT_TAG        fix_laplace_box
```
Comment (StRigaud) [inline thirdparty/CMakeLists.txt:8]: Kernel is pushed to the clesperanto_kernel branch, you can update back the repository link and tag before final merge

**Review by StRigaud — APPROVED** [review]

**Pull request merged**